src/channels/pages/ChannelDetailsPage/ChannelDetailsPage.tsx

```diff
@@ -165,6 +186,45 @@ const ChannelDetailsPage = function<TErrors>({
           );
         };
 
+        const addWarehouse = (warehouseId: string) => {
```

Comment:
`triggerChange` or something because exit dirty form confirmation is not working. Same for remove warehouse.